### DISCUSSION TITLE
[MARKETING REQUEST] GP2040-CE Logo usage

### DISCUSSION THREAD

**Discussion by NUMB41** [OpenStickCommunity/GP2040-CE#412] [Q&A]
Dear GP2040-CE,

I am creating a fan book for Comic Market.
We would like to use the GP2040-CE logo in this magazine.
I want Japanese fighting gamers to know more about it.
Could you please accept my request?

Although it is a draft version, we will send you a fan book.

![GP2040-ce check](https://github.com/OpenStickCommunity/GP2040-CE/assets/78263959/4aebcf00-8f8d-40f5-a45c-f5633805dbed)

**Comment by bsstephan** [comment]
Hi @NUMB41. Cool! I passed this on to @TheTrainGoes and there'll be an update/clarification to the assets and their usage coming soon.

**Comment by TheTrainGoes** [comment]
Hi @NUMB41 ,

Very late on this but we have the proper licence now and you can find assets in the hardware repo.
